crates/ruff/src/printer.rs

```diff
@@ -417,6 +417,13 @@ impl Printer {
                         statistic.name,
                     )?;
                 }
+
+                if any_fixable {
+                    writeln!(
+                        writer,
+                        "[*] indicates that a violation is fixable with the `--fix` option."
```

Comment:
I would use something closer to the `ruff check` message
```suggestion
                        "[*] fixable with `ruff check --fix`
```

Original message:

```
[*] 30 fixable with the `--fix` option (31 hidden fixes can be enabled with the `--unsafe-fixes` option).
```